README.md

```diff
@@ -0,0 +1,77 @@
+This is a dashboard for tracking outstanding bugs in **Ironic**.
+
+[**Ironic**](https://ironicbaremetal.org/) is one of the software components in OpenStack that fully manages bare-metal infrastructure; it discovers bare-metal nodes; catalogs them in a management database and manages the entire server lifecycle; from enrolling to provisioning, maintenance and decommissioning.
+
+[**OpenStack**](https://www.openstack.org/) is an opensource cloud software project that includes a collection of software components providing common services for cloud infrastructure.
+
+## Setup
+Before diving into the setup process, ensure that your Python version is `>=python3.11`.
+
+### Creating a Virtual Environment
```

Comment:
I think we explicitly do not want to encourage this and instead direct people to use `tox` as the basic runner. If someone wants to do a manual venv; they can use documentation somewhere else. Putting too much specifics here (e.g. full details on how to build a venv) makes this document more likely to need revision in the future.